version.txt

```diff
@@ -1 +1 @@
-0.2.9.20240419
+0.2.9.20240422
```

Comment:
Minor version should be bumped too because you've changed only implementation details.

Reply:
Can you please help me with this? I don't think I understand this completely.

Reply:
Sure @alexprabhat99. You'll want to follow the process [here](https://github.com/pistacheio/pistache/?tab=readme-ov-file#versioning). It's just standard semantic versioning.

If you changed existing public interfaces you bump the major version. If you only added to the existing public interfaces, but did not change them, then you bump the minor version. If you didn't add or change any existing public interfaces, but changed something internally in the implementation, you bump the patch version.

This ensures the linker generates the correct library soname so that user code doesn't link when it shouldn't and still does when it should.

In your case I think you want 0.2.10.20240422.

Reply:
Okay, got it. Thanks!